src/core/tests/type_prop/convolution_backprop_data.cpp

```diff
@@ -822,3 +823,18 @@ TEST(type_prop, convolution_backprop_data_invalid_conv_param_spatial_dims) {
         FAIL() << "Output padding spatial dimensions validation check failed for unexpected reason";
     }
 }
+
+TEST(type_prop, convolution_back_prop_data_default_constructed) {
+    auto conv = make_shared<op::v1::ConvolutionBackpropData>();
+
+    const auto &input_shape = ov::PartialShape::dynamic(), filters_shape = ov::PartialShape{1, 1, 3, 3},
+               output_spatial_shape_shape = ov::PartialShape({2});
+    const auto& input_shapes = std::vector<ov::PartialShape>{input_shape, filters_shape, output_spatial_shape_shape};
+    std::vector<ov::PartialShape> output_shapes = {ov::PartialShape::dynamic()};
+    auto pad_begin = CoordinateDiff{}, pad_end = CoordinateDiff{};
+    const auto& output_spatial_shape = ov::PartialShape{3, 3};
+    int64_t num_spatial =
+        calculate_num_spatial(conv.get(), input_shape, filters_shape, output_spatial_shape_shape, 2, 2);
+    update_and_validate_attributes_back_prop(conv.get(), num_spatial);
+    shape_infer(conv.get(), pad_begin, pad_end, output_spatial_shape, input_shapes, output_shapes);
```

Comment:
Missing test assert/except macro?
Check other similar added test it looks there is no `gtest` expectation macro.